src/main/java/frc/team2412/robot/Subsystems.java

```diff
@@ -11,7 +11,7 @@
 
 public class Subsystems implements Loggable {
     public static class SubsystemConstants {
-        public static final boolean CLIMB_ENABLED = true;
+        public static final boolean CLIMB_ENABLED = false;
```

Comment:
I imagine we still expect to be able to climb at least to mid bar for competition so we probably don't want this change in.